apis/v1beta1/shared_types.go

```diff
@@ -104,6 +122,10 @@ type ParentReference struct {
 	// and SectionName are specified, the name and port of the selected listener
 	// must match both specified values.
 	//
+	// When the parent resource is a Service, this targets a specific port in the
+	// Service spec. When both Port (experimental) and SectionName are specified,
```

Comment:
What does SectionName mean for a Service?

Reply:
SectionName means Port name, will clarify.

Reply:
Covered in #2261